File: 프로그래머스/0/120852. 소인수분해/소인수분해.js
function solution(n) {
    var answer = [];
    for (let i = 2; i <= n; i++) {
        while (n % i === 0) {
            n = n / i
            if (n % i !== 0) {
                answer.push(i)
            }
        }
    }
    return answer;
}

File: 프로그래머스/0/120852. 소인수분해/소인수분해.py
def solution(n):
    answer = []
    i = 2
    while i * i <= n:
        if n % i == 0:
            answer.append(i)
            while n % i == 0:
                n //= i
        else:
            i += 1
    if n > 1:
        answer.append(n)
    return answer